src/components/ConsumptionGraph.js

```diff
@@ -44,7 +44,7 @@ class ConsumptionGraph extends Component {
                             zoomin: false,
                             zoomout: false,
                             pan: false,
-                            reset: true | '<img src="/static/icons/reset.png" width="20">',
+                            reset: true  | '<img src="/static/media/tree.svg" width="40">',
```

Comment:
This is the reset icon of the consumption graph, right? Why the change to the tree icon of the consumption calculation section?

Reply:
Oh sorry! I was testing if the custom icons worked (they don't) and totally forgot to remove. I#ll remove and push again.

Reply:
Done.